lib/SPIRV/SPIRVToOCL12.cpp

```diff
@@ -91,6 +96,24 @@ void SPIRVToOCL12::visitCallSPIRVMemoryBarrier(CallInst *CI) {
       &Attrs);
 }
 
+void SPIRVToOCL12::visitCallSPIRVControlBarrier(CallInst *CI) {
+  AttributeList Attrs = CI->getCalledFunction()->getAttributes();
+  Attrs = Attrs.addAttribute(CI->getContext(), AttributeList::FunctionIndex,
+                             Attribute::NoDuplicate);
+  CallInst *NewCI = mutateCallInstOCL(
```

Comment:
Will remove variable assignment from here, as NewCI is unused.

Reply:
Are you still going to remove this variable assignment?